src/shabdawali.js

```diff
@@ -231,20 +231,20 @@ Shabdawali.prototype.typeText = function(cLine){
                 //this.element.textContent  += char;
                 this.element.textContent  = cLine.substr( 0, this.currentLetterIndex  );
                 var that = this;
-                if(char == ' ')
-                {
+                if(char == ' ') {
```

Comment:
we need to check for user options `if( this.dynamicPause && char === ' ' )`. 'this.dynamicPause' need to be set in constructor.